core/src/util.jl

```diff
@@ -750,3 +750,27 @@ function get_basin_idx(edge_metadata::EdgeMetadata, id::NodeID)::Int32
         0
     end
 end
+
+function set_initial_allocation_mean_flows!(integrator)::Nothing
```

Comment:
Could use a comment that we don't have mean flows at t=0, but use the instantaneous flow at t=0 as an approximation.